changelog/56864.fixed

```diff
@@ -0,0 +1 @@
+Display packages that are marked NoRemove
```

Comment:
Can you elaborate here. What modules does this impact?

Maybe:

Display packages that are marked NoRemove in the pkg.list_pkgs for windows platforms.

Reply:
Done